requirements.txt

```diff
@@ -6,7 +6,7 @@ colander
 cornice>=3.1.0
 dogpile.cache
 pyasn1-modules  # Due to an unfortunate dash in its name, installs break if pyasn1 is installed first
-fedora_messaging
+fedora_messaging>=1.6.0
 feedgen
 jinja2
 kitchen
```

Comment:
Since we directly use twisted now, we should also add twisted to this list.

Reply:
Right, I've added it.